DNN Platform/Website/Install/Config/09.06.02.config

```
@@ -0,0 +1,7 @@
+<configuration>
+  <nodes configfile="Web.config">
+    <node path="/configuration/system.web/staticContent/ action="update" key="fileExtension" collision="ignore">
```

```suggestion
    <node path="/configuration/system.web/staticContent/" action="update" key="fileExtension" collision="ignore">
```

There was a missing double-quote that made install/upgrades fail with invalid xml. I will accept my own change here and let it build and try again...